Core/ELF/ElfReader.cpp

```diff
@@ -413,6 +413,11 @@ int ElfReader::LoadInto(u32 loadAddress, bool fromTop)
 		}
 	}
 	totalSize = totalEnd - totalStart;
+
+	// If a load address is specified that's in regular RAM, override kernel module status
+	bool inUser = totalStart > 0x08900000;
```

Comment:
`>= PSP_GetUserMemoryBase()` might be preferrable I guess, not sure what 0x08900000 indicates.

-[Unknown]